hedera-node/hedera-addressbook-service-impl/src/main/java/module-info.java

```diff
@@ -12,8 +12,8 @@
     requires com.hedera.node.app.service.token;
     requires com.hedera.pbj.runtime;
     requires org.apache.logging.log4j;
+    requires static transitive java.compiler;
```

Comment:
JavaCompile's `export` check reveals that this should be transitive, because the `@Generated` annotation (located in the `java.compiler` module) is used to annotate public API.